SPECS/xapi.spec

```diff
@@ -79,7 +79,7 @@ BuildRequires: python2-devel
 %endif
 
 BuildRequires: python3-devel
-BuildRequires: xs-opam-repo >= 6.54.0-2
+BuildRequires: xs-opam-repo > 6.74.0-1.1
```

Comment:
It's a strictly superior operator, is it wanted?
Especially since you mention this exact version in the changelog comment.

Reply:
This is intended for the testing phase, while `6.74.0-1.2` is not issued yet.  But `yum-builddeps` happily ignores it, so we'd better prepare for it already.

Reply:
The intention is good, but this line will conflict as soon as XenServer updates it and I know I'll likely be the one handling the conflict :)

I'll opt for a neutral approach in my review regarding this line.

Reply:
done